docs/examples/example_figure.py

```diff
@@ -0,0 +1,83 @@
+from matplotlib import pyplot as plt
+from matplotlib import colors as colors
+import numpy as np
+from astropy import units as u
+from screens.fields import dynamic_field
+from screens.secspec import SecondarySpectrum
+from screens.visualization import make_sketch
+
+
+def gen_magnifs(th):
```

Comment:
Nitpick, but since in `dynspec` and `secspec`, this is called `magnification`, maybe just stick to that? Maybe `generate_magnification`

Reply:
Done